src/xcode/ENA/ENA/Source/Services/PPAnalyticsSubmitter/PPAnalyticsSubmitter.swift

```diff
@@ -89,6 +89,8 @@ final class PPAnalyticsSubmitter: PPAnalyticsSubmitting {
 			} else {
 				self.generatePPACAndSubmitData(completion: completion)
 			}
+			
+			self.hoursSinceTestRegistrationToSubmitTestResultMetadata = configuration.privacyPreservingAnalyticsParameters.common.hoursSinceTestRegistrationToSubmitTestResultMetadata
```

Comment:
should we store the var at the beginning of all the checks or does the tech spec wanted it as you implemented it?

Reply:
it says:
"the difference in hours between the timestamp of test registration and the current device time is greater than or equal to the value of Configuration Parameter .privacyPreservingAnalyticsParameters .common .hoursSinceTestRegistrationToTestResultMetadata."

So I think whenever we fetch a new appconfig we should update this duration